test/com/ableton/PyenvTest.groovy

```diff
@@ -133,8 +130,8 @@ class PyenvTest extends BasePipelineTest {
     helper.registerAllowedMethod('error', [String]) { errorCalled = true }
     helper.registerAllowedMethod('fileExists', [String]) { return true }
     helper.registerAllowedMethod('isUnix', []) { return true }
+    helper.addShMock("${pyenvRoot}/bin/pyenv install --list", '1.0.0', 0)
     helper.addShMock("${pyenvRoot}/bin/pyenv --version", 'pyenv 1.2.3', 0)
-    helper.addShMock(installCommands(pyenvRoot, pythonVersion), '', 1)
```

Comment:
In abe5d0880b2e7792609b7219c6218663cd52d845, if you remove this, then there is no exception thrown below and the test fails.